sentry-android-core/src/main/java/io/sentry/android/core/SystemEventsBreadcrumbsIntegration.java

```diff
@@ -206,10 +216,43 @@ public void onReceive(final Context context, final @NotNull Intent intent) {
                   scopes.addBreadcrumb(breadcrumb, hint);
                 });
       } catch (Throwable t) {
-        options
-            .getLogger()
-            .log(SentryLevel.ERROR, t, "Failed to submit system event breadcrumb action.");
+        // ignored
+      }
+    }
+
+    // in theory this should be ThreadLocal, but we won't have more than 1 thread accessing it,
+    // so we save some memory here and CPU cycles. 64 is because all intent actions we subscribe for
+    // are less than 64 chars. We also don't care about encoding as those are always UTF.
+    // TODO: _MULTI_THREADED_EXECUTOR_
+    private final char[] buf = new char[64];
+
+    @TestOnly
```

Comment:
is this `@TestOnly` wanted?
The code is used normally, so i would either use VisibleForTesting or Internal

Reply:
It's package-private, so I added TestOnly so it gets warned if it's used in a different class. Otherwise could've been private but yeah we need it for tests